### DISCUSSION TITLE
Has anyone gotten benefit out of caption_dropout_rate?

### DISCUSSION THREAD

**Discussion by zhaol4v** [kohya-ss/sd-scripts#1804] [General]
If so, what value did you use? I've tried 5%, 10%, tried training without any txt files....the result has never been better for me than using full captions

**Comment by rockerBOO** [comment]
It can help a little with flexibility of the captions and prompting but if too much it can risk overfitting more. This is due to the lack of guidance from the captions skews the loss calculations. I find caption_tag_dropout_rate and shuffle_captions more effective balancing point. Each dataset, training, and goals will need to be evaluated to find what balance works for you.
